Build or publish script seems to be failing

First, thank you so much for rollup @Rich-Harris !

I just installed this plugin and got this in my package.json:

```
"rollup-plugin-strip": "^1.0.2"
```

And this in my ```node_modules``` folder:

```
$ ll node_modules/rollup-plugin-strip/
total 16
-rw-r--r-- 1 1000 staff  182 Jun  9 15:22 CHANGELOG.md
drwxr-xr-x 1 1000 staff  136 Jun  9 23:04 node_modules
-rw-r--r-- 1 1000 staff 2816 Jun  9 23:03 package.json
-rw-r--r-- 1 1000 staff  744 Jun  3 15:25 README.md
-rw-r--r-- 1 1000 staff  108 Jun  3 13:35 rollup.config.js
drwxr-xr-x 1 1000 staff  102 Jun  9 15:22 src
drwxr-xr-x 1 1000 staff  136 Jun  9 23:03 test
```

No ```dist/``` folder. So, I guess that's why I got the last message when trying to use it in my _rollup.config.js_ file:

```
Cannot find module 'rollup-plugin-strip'
Error: Cannot find module 'rollup-plugin-strip'
    at Function.Module._resolveFilename (module.js:339:15)
```

It seems the build and publish package is not working properly.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,0 +1,5 @@
+language: node_js
+node_js:
+  - "4.0"
+  - "5.0"
+  - "6.0"

--- a/package.json
+++ b/package.json
@@ -36,6 +36,7 @@
   "devDependencies": {
     "eslint": "^2.11.1",
     "mocha": "^2.5.3",
+    "rollup": "^0.30.0",
     "rollup-plugin-buble": "^0.10.0"
   }
 }

--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
   "version": "1.0.2",
   "description": "Remove debugger statements and functions like assert.equal and console.log from your code.",
   "main": "dist/rollup-plugin-strip.cjs.js",
+  "jsnext:main": "dist/rollup-plugin-strip.es.js",
   "scripts": {
     "test": "mocha",
     "pretest": "npm run build:cjs",
@@ -38,5 +39,8 @@
     "mocha": "^2.5.3",
     "rollup": "^0.30.0",
     "rollup-plugin-buble": "^0.10.0"
-  }
+  },
+  "files": [
+    "dist"
+  ]
 }

--- a/package.json
+++ b/package.json
@@ -6,12 +6,10 @@
   "jsnext:main": "dist/rollup-plugin-strip.es.js",
   "scripts": {
     "test": "mocha",
-    "pretest": "npm run build:cjs",
-    "build": "npm run build:cjs && npm run build:es",
-    "build:cjs": "rollup -c -f cjs -o dist/rollup-plugin-strip.cjs.js",
-    "build:es": "rollup -c -f es6 -o dist/rollup-plugin-strip.es.js",
+    "pretest": "npm run build",
+    "build": "rollup -c",
     "lint": "eslint src",
-    "prepublish": "npm run lint && npm test && npm run build:es"
+    "prepublish": "npm run lint && npm test"
   },
   "repository": {
     "type": "git",

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -2,5 +2,15 @@ import buble from 'rollup-plugin-buble';
 
 export default {
 	entry: 'src/index.js',
-	plugins: [ buble() ]
+	plugins: [ buble() ],
+	targets: [
+		{
+			format: 'cjs',
+			dest: 'dist/rollup-plugin-strip.cjs.js'
+		},
+		{
+			format: 'es6',
+			dest: 'dist/rollup-plugin-strip.es.js'
+		}
+	]
 };

